README.md

```diff
@@ -1,7 +1,7 @@
-Jenkins Enterprise by CloudBees Plugin
+CloudBees Jenkins Enterprise Plugin
 ======================================
 
-Read more: [https://wiki.jenkins-ci.org/display/JENKINS/Jenkins+Enterprise+by+CloudBees](https://wiki.jenkins-ci.org/display/JENKINS/Jenkins+Enterprise+by+CloudBees)
+Read more: https://wiki.jenkins-ci.org/display/JENKINS/CloudBees+Jenkins+Enterprise
```

Comment:
> Does our Confluence version support links to moved pages?

According to [CONF-2906](https://jira.atlassian.com/browse/CONF-2906), no, yet the old page is still rendered, I think by the custom caching reverse proxy, but it cannot be _edited_, so I cannot set up a manual redirect. Sigh.